staging/src/k8s.io/client-go/util/workqueue/queue_test.go

```diff
@@ -159,3 +159,52 @@ func TestReinsert(t *testing.T) {
 		t.Errorf("Expected queue to be empty. Has %v items", a)
 	}
 }
+
+func TestAddMulti(t *testing.T) {
+	q := workqueue.New()
+
+	// Start consumers
+	seen := sync.Map{}
+	const consumers = 10
+	consumerWG := sync.WaitGroup{}
+	consumerWG.Add(consumers)
+	for i := 0; i < consumers; i++ {
+		go func(i int) {
+			defer consumerWG.Done()
+			for {
+				item, quit := q.Get()
+				if quit {
+					return
+				}
+				_, loaded := seen.LoadOrStore(item, true)
+				if loaded {
+					t.Errorf("Consumer saw item %v again", item)
```

Comment:
`"Consumer %d saw item %v again"`

Reply:
Would that help? The ordering is non-deterministic so unless I also log a full trace of all operations, the consumer ID wouldn't help with debugging.

Reply:
then `i int` is unnecessary as a function parameter.

Reply:
I'm not sure we need to get into micro-optimizations of test code, but sure.